Helm: add configuration options for kvstoremesh (#26109)
cilium-team/cilium: Documentation/observability/metrics.rst

Installation
------------

> You can use the following settings to enable the metrics for the different

Reviewer: nit: settings -> values?

Installation
------------

> You can use the following values to enable the metrics for the different
> Cluster Mesh API Server components:

Reviewer 2: Edits for clarity
```suggestion
You can enable the metrics for different
Cluster Mesh API Server components by setting the following values:
```

* clustermesh-apiserver: ``clustermesh.apiserver.metrics.enabled=true``
* kvstoremesh: ``clustermesh.apiserver.metrics.kvstoremesh.enabled=true``
* sidecar etcd instance: ``clustermesh.apiserver.metrics.etcd.enabled=true``

.. parsed-literal::

   helm install cilium |CHART_RELEASE| \\
     --namespace kube-system \\
     --set clustermesh.useAPIServer=true \\
     --set clustermesh.apiserver.metrics.enabled=true \\
     --set clustermesh.apiserver.metrics.kvstoremesh.enabled=true \\
     --set clustermesh.apiserver.metrics.etcd.enabled=true

> The ports can be configured via ``clustermesh.apiserver.metrics.port``,

Reviewer 2: Use active voice, no Latin abbreviations
```suggestion
You can figure the ports by way of ``clustermesh.apiserver.metrics.port``,
```
